src/intro/introduction.md

```diff
@@ -24,8 +28,13 @@ particular architecture and explains details particular to this architecture
 where required.
 
 ## Who This Book is For
+This book caters towards people with either a solid embedded background or a solid rust background, however we assume
```

Comment:
"solid" is quite a strong word IMHO, I'd tone this down a little bit so people don't give up at this point already. How about "some"? Also the repetition of "solid" and "background" could be avoided.

Reply:
I'll change that, still leaving out one some/solid sounds kinda odd to me (without restructuring the sentence).